libs/sdk-ui-dashboard/src/presentation/dashboard/DashboardHeader/ShareDialogDashboardHeader.tsx

```diff
@@ -59,6 +61,13 @@ const useShareDialogDashboardHeader = () => {
         addError(messages.messagesSharingDialogError);
     }, [dispatch, addError]);
 
+    const onInteractionShareDialog = useCallback(
+        (data: ShareDialogInteractionData) => {
+            shareDialogInteraction(data);
+        },
+        [shareDialogInteraction],
+    );
```

Comment:
Why to cache this? Couldn't we just pass it down as provided?

Reply:
This must be a remain of my first attempts where I had different dependencies.. Will remove, thanks